grafana/dashboards/7mode/harvest_dashboard_network7.json

```diff
@@ -816,6 +790,30 @@
                 "value": 398
               }
             ]
+          },
```

Comment:
`calculateField` is missing from 7-mode dashboard

Reply:
added